source/MaterialXGraphEditor/Main.cpp

```diff
@@ -30,6 +30,7 @@ const std::string options =
     "    --path [FILEPATH]              Specify an additional data search path location (e.g. '/projects/MaterialX').  This absolute path will be queried when locating data libraries, XInclude references, and referenced images.\n"
     "    --library [FILEPATH]           Specify an additional data library folder (e.g. 'vendorlib', 'studiolib').  This relative path will be appended to each location in the data search path when loading data libraries.\n"
     "    --captureFilename [FILENAME]   Specify the filename to which the first rendered frame should be written\n"
+    "    --scaleFactor [FACTOR]         Manually specify a HiDPI scaling factor\n"
```

Comment:
The substance of this change looks good to me, @virokannas, and my one recommendation would be to rename this command-line option from `scaleFactor` to `dpiScale` for improved user clarity.

Reply:
I can do that @jstone-lucasfilm if you want - but wouldn't having the word "dpi" in the option name hint towards having to use a DPI value or a DPI-dependent value rather than a scaling factor? The actual variable being driven by this is a factor that is unaware of the screen DPI.

I'm sure at least some who would use this are also familiar with QT, which has the same setting labeled as QT_SCALE_FACTOR. Linux window managers call it a scaling factor, with varying ranges, Windows uses percentages, etc..

But if dpiScale fits better within the MaterialX ecosystem, I'll change it no problem.

Reply:
@virokannas That's a good point, and I'm open to better ideas for terminology here.  In the ImGui documentation, they provide the following definition of ImGuiStyle::ScaleAllSizes, which is the method that is influenced by this command-line option.

>// To scale your entire UI (e.g. if you want your app to use High DPI or generally be DPI aware) you may use this helper function. Scaling the fonts is done separately and is up to you.

Mainly, I'd like to clarify that this option affects UI/DPI scaling, and is not related to the scale of the viewport render.

Reply:
@jstone-lucasfilm Ah, of course.

How about "uiScale" or "uiScaleFactor"?

Reply:
Sure, let's go with `uiScale` to keep things simple.